overrides/android-override.json

```diff
@@ -546,7 +546,29 @@
                                 "featuresLogged": {
                                     "gpc": true,
                                     "cpm": true,
-                                    "appTp": false,
+                                    "appTP": false,
+                                    "netP": false
+                                }
+                            },
+                            {
+                                "appVersion": 52220000,
+                                "daysAfterBuild": 6,
```

Comment:
since this is the last release of the year, we can have daysLogging to a bigger number, like 28. We won’t cut a new release until January 6th, so leaving it to 10 is wasting days of data. What do you think?

with this numbers, as soon as this is merged and users start getting the new config we start logging.

Reply:
Thanks, makes sense :+1: Updated.